Mail/Components/AvatarView.swift

```diff
@@ -52,5 +70,6 @@ struct AvatarView: View, Equatable {
             }
         }
         .accessibilityLabel(MailResourcesStrings.Localizable.contentDescriptionUserAvatar)
+        .id(id)
```

Comment:
I would put directly `viewModel.displayablePerson.id` here but even then I don't think this would be right.
The id should be on the `AvatarView` to tie it to the displayed contact.

Reply:
Ok to be more in the spirit of swiftUI I can make this ID explicit with `viewModel.displayablePerson.id`

Reply:
Not needed in the end.